js/src/common/components/Modal.tsx

```diff
@@ -124,12 +124,16 @@ export default abstract class Modal<ModalAttrs = {}> extends Component<ModalAttr
   /**
    * Get the content of the modal.
    */
-  abstract content(): Mithril.Children;
+  content(): Mithril.Children {
```

Comment:
Shouldn't we require `content` of some form?

Reply:
`content` is called inside the form which is inside the default view, when you're implementing a non form modal, you'll implement the view method because of that. you won't need the default form JSX in the default view.

Reply:
oh wait no, you're right, you can't and shouldn't be reimplementing the view method, you're have to recreate the base jsx, content needs to be required, but that also means even if it isn't a form modal it'll be within a form, which isn't logical.

Something to fix in 2.0, I'll re-require the content method